src/main/java/crazypants/enderio/material/MaterialRecipes.java

```diff
@@ -60,6 +60,18 @@ public static void registerOresInDictionary() {
     OreDictionary.registerOre("sand", new ItemStack(Blocks.sand, 1, OreDictionary.WILDCARD_VALUE));
     OreDictionary.registerOre("ingotIron", Items.iron_ingot);
     OreDictionary.registerOre("ingotGold", Items.gold_ingot);
+	OreDictionary.registerOre("blockIron", Blocks.iron_block);
```

Comment:
I know that these in Forge. I was wondering why ingotIron, ingotGold and etc registered here, this is why I've added new ones. Should I remove those that I've added? And just add craftingTableWood and chestWood to [OreDict.java](https://github.com/SleepyTrousers/EnderCore/blob/master/src/main/java/com/enderio/core/common/OreDict.java)?